themes/default/content/blog/announcing-pulumiup-conference-2024/index.md

```diff
@@ -0,0 +1,78 @@
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2024-05-01T17:27:25Z
+
+# The draft setting determines whether a post is published. Set it to true if
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: PulumiUP 2024 is back! Register to attend on September 18, and dive into the future of cloud, Infrastructure as Code (IaC), AI, platform engineering, and more.
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: pulumiup-2024-announcement.png
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - sara-huddleston
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - pulumi-events
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+
+Today we announce our annual PulumiUP virtual conference taking place on September 18, 2024. PulumiUP is our global cloud and IaC conference for anyone interested in infrastructure as code, cloud, AI, platform engineering, DevOps, and cloud culture. Every year, we strive to build a program packed with technical talks, demos, and best practices to empower you with new learnings and inspiration that you can apply in your professional life. Attend to hear from industry leaders, experts, and community peers.
+
+<!--more-->
+
+## What is PulumiUP?
+
+PulumiUP isn't just another conference, it's a global gathering of minds passionate about cloud technologies, AI, DevOps, and everything in between. Whether you're an industry veteran or just dipping your toes into the vast ocean of cloud computing, PulumiUP has something for everyone.
+
+You will hear keynote from Pulumi leaders, Founder/CEO Joe Duffy and CTO Luke Hoban, and from industry leaders on their experience in cloud adoption, innovation and transformation and how they are leverage Pulumi to make it happen. We will have special industry expert speakers to help you navigate the everchanging world of the cloud and hear from the community, which can be you if you have a story to tell that brings value to cloud ecosystem community.
```

Comment:
This feels like it could use a little tightening up. Maybe something like this?

```suggestion
You'll hear keynotes from Pulumi Founder/CEO Joe Duffy and CTO Luke Hoban, stories from industry leaders on their experiences with cloud adoption and transformation (and how they used Pulumi to make it happen), and talks from other members of the Pulumi community (like you!) aimed at helping you navigate the ever-changing world of the cloud.
```